Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'azure_mgmt_dns',                       path: 'management/azure_mgmt_dns'
 gem 'azure_mgmt_event_hub',                 path: 'management/azure_mgmt_event_hub'
 gem 'azure_mgmt_features',                  path: 'management/azure_mgmt_features'
 gem 'azure_mgmt_graph',                     path: 'management/azure_mgmt_graph'
-gem 'azure_mgmt_insights',                  path: 'management/azure_mgmt_insights'
+gem 'azure_mgmt_monitor',                   path: 'management/azure_mgmt_monitor'
```

Comment:
IMO, with this change we should encourage customers to move to `azure_mgmt_monitor` gem if they are using `azure_mgmt_insights` or provide smooth transition / migration guidelines. Just curious to know Do we have a plan for this & how? May be something like marking existing gem deprecate warning message on usage etc... 

@veronicagg @devigned Thoughts?